### ISSUE TITLE
Table of Contents has broken links

### ISSUE THREAD

**Issue by LoganTFox** [dniccum/nova-documentation#28]
When I try to click a link on the side card table of contents, the links are broken and it brings me back to the main dashboard of Nova.

<img width="382" alt="Screen Shot 2023-02-23 at 1 42 09 PM" src="https://user-images.githubusercontent.com/31284288/221013075-7247bc14-ffcb-47e8-bfb9-4363db6b7841.png">
<img width="228" alt="Screen Shot 2023-02-23 at 1 42 38 PM" src="https://user-images.githubusercontent.com/31284288/221013171-8b5cd3ba-2acb-4e94-a831-ae97ea7d164d.png">

**Comment by dniccum** [comment]
This is very likely related to #26. Assuming it is, this is a known issue. I am working on a fix.

**Comment by LoganTFox** [comment]
Sounds good thank you! I appreciate your speedy work!

**Comment by mstaack** [comment]
yeah seeing the same issue. basically a wrong created link resulting in a 404 route call like this
```
All Inertia requests must receive a valid Inertia response, however a plain JSON response was received.
{"message":"Not found"}
```

proposed a change that fixed my issue in #33
